File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    result=[chr(ord(c)+index) for c in s]
    
    for c in s:
        i=1
        change=c
        while i<=index:
            change=chr(ord(change)+1)
            if change>'z':
                change='a'
            if change not in skip:
                i+=1
                
        answer+=change
                
        
    return answer

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.java
import java.util.*;
class Solution {
    public String solution(String s, String skip, int index) {
        String answer = "";
        boolean[] no=new boolean[26];
        for(char c:skip.toCharArray()){
            no[c-'a']=true;
        }
        for(char c:s.toCharArray()){
            char al=(char)(c+1);
            int i=0;
            while(i<index){
                if(!no[(al-'a')%26]){
                    i++;
                }  
                System.out.println(al);
                al=(char)(al+1);
                
            }
         
            char after=(char)((al-1-'a')%26+'a');
            answer+=after;
        }
           return answer;
           
          
        }
       
       
    
}